free-programming-books.md

```diff
@@ -403,6 +403,7 @@
 * [Computational Thinking](https://www.cs.cmu.edu/~15110-s13/Wing06-ct.pdf) - Jeannette Wing, Carnegie-Mellon University (PDF)
 * [Computer Science I - Draft](https://bitbucket.org/chrisbourke/computersciencei/src) - Dr. Chris Bourke (PDF) (:construction: *in process*)
 * [Computer Science II - Draft](http://cse.unl.edu/~cbourke/ComputerScienceTwo.pdf) - Dr. Chris Bourke (PDF) (:construction: *in process*)
+* [CS Principles: Big Ideas in Programming](http://www.openbookproject.net/books/StudentCSP/index.html#) - Mark Guzdial-Barbara Ericson (HTML)
```

Comment:
It appears this page supports HTTPS. Could you please use an https:// instead?

Also can you remove `index.html#` from the end as this is redundant.

For example: `https://www.openbookproject.net/books/StudentCSP/`

Reply:
+1